components/script/dom/domrectlist.rs

```diff
@@ -57,3 +66,18 @@ impl DOMRectListMethods for DOMRectList {
         self.Item(index)
     }
 }
+
+impl<'a> Iterator for Iter<'a> {
```

Comment:
@KiChjang I though that but it would mutate the DOMRectList and it would only be iterated once because of `position`. Does it make sense?
@ConnorGBrewster I used the same naming as in the std, is better to change it?